plugins/ltac/g_obligations.mlg

```diff
@@ -78,10 +78,8 @@ GRAMMAR EXTEND Gram
 
 {
 
-open Obligations
```

Comment:
Why not open Declare.Obls here for a smaller diff?

Reply:
Done, thanks.